api/internal/handler/route/route_test.go

```diff
@@ -925,6 +927,42 @@ func TestRoute(t *testing.T) {
 	dataPage = retPage.(*store.ListOutput)
 	assert.Equal(t, len(dataPage.Rows), 0)
 
+	// list search and status match
+	listInput = &ListInput{}
+	reqBody = `{"page_size": 1, "page": 1, "status": "1"}`
+	err = json.Unmarshal([]byte(reqBody), listInput)
+	assert.Nil(t, err)
+	ctx.SetInput(listInput)
+	retPage, err = handler.List(ctx)
+	assert.Nil(t, err)
+	dataPage = retPage.(*store.ListOutput)
+	assert.Equal(t, len(dataPage.Rows), 1)
+
+        //sleep
+        time.Sleep(time.Duration(100) * time.Millisecond)
```

Comment:
Style: redundant space.

Reply:
Hi, @Jaycean don't forget to fix the style problem.

Reply:
done.